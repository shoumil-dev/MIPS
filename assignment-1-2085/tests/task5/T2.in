5
10
40
80
30
10
4
