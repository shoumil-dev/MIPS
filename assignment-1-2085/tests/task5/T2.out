MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar

Array length: Enter num: Enter num: Enter num: Enter num: Enter num: Enter r: 10 40 80 30 
10 40 80 10 
10 40 30 10 
10 80 30 10 
40 80 30 10 

